content/projects/_index.md

```diff
@@ -4,9 +4,75 @@ description: "This page provides the details of SODA Projects"
 weight: 4
 disableToc: false
 ---
-[SODA Foundation](https://sodafoundation.io) is a home for multiple projects for building a unified data framework to support various data and storage solutions. All the projects maintained by SODA Foundation will be described under this documentation. All other projects part of SODA Foundation ecosystem, please refer to the respective project page.
+[SODA Foundation](https://sodafoundation.io) is a home for multiple projects for building a unified data framework to support various data and storage solutions. Here is a brief description of all the projects maintained by SODA Foundation. 
 
-## Projects maintained by SODA Foundation
+<!-- Soda Project Architecutre Diagram -->
+![SODA Projects Architecture](soda-project-architecture.png)
+
+- ### SODA Dashboard 
```

Comment:
nit: instead of making this a list with bullets just the headings are enough.

Reply:
![image](https://user-images.githubusercontent.com/65751939/91690308-5951d600-eb83-11ea-94fd-7ac1c8fa2335.png)
With links we can show the sub headings on the page menu